### PR TITLE
Lapack detect

### PR DESCRIPTION
**Pull request by ipelupessy**
it seems to change (too) much..

### CODE REVIEW

**Comment by ipelupessy** [comment]
e.g. the shtool needs to be there (or install-sh), but is not used..

**Comment by ipelupessy** [comment]
I am not sure about this last thing; the Makefile can also be used to build target that do not depend on configure having run (like releases) but I think most of these are broken and should maybe not be in the Makefile to begin with...(so we can clean this up)

**Comment by ipelupessy** [comment]
@rieder  comments?

**Comment by ipelupessy** [comment]
I think this can be merged..

**Comment by ipelupessy** [comment]
@rieder do you think this is ok? its mostly harmless but for the Makefile: it is now generated after configure, which makes sense but means that you cannot "do things" before that...

**Comment by rieder** [comment]
Yeah this is fine

File: Makefile.in [hunk 1]
```diff
@@ -46,7 +40,6 @@ clean:
 
 oclean:
```
Comment (rieder) [inline Makefile.in:41]: I see most of the omuse things are removed, isn't this also part of that?

Comment (ipelupessy) [reply Makefile.in:41]: yea, OMUSE is a completely independent package now, with its own setup.py..(and only depends on amuse-framework)

File: config.guess [hunk 2]
```diff
@@ -0,0 +1,1700 @@
+#! /bin/sh
+# Attempt to guess a canonical system name.
+#   Copyright 1992-2021 Free Software Foundation, Inc.
+
+timestamp='2021-01-25'
```
Comment (rieder) [inline config.guess:5]: fixed timestamp?

File: config.sub [hunk 3]
```diff
@@ -0,0 +1,1860 @@
+#! /bin/sh
+# Configuration validation subroutine script.
+#   Copyright 1992-2021 Free Software Foundation, Inc.
+
+timestamp='2021-03-10'
```
Comment (rieder) [inline config.sub:5]: also here, fixed timestamp?

File: configure [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.69 for amuse 2021.6.1.dev0+g380eaa363.d20210624.
+# Generated by GNU Autoconf 2.69 for amuse 2021.4.1.dev10+g50bc12e1e.d20210920.
```
Comment (rieder) [inline configure:3]: maybe autoconf should be re-run

Comment (ipelupessy) [reply configure:3]: yea

File: config.sub (hunk 3 above)
Comment (ipelupessy) [reply config.sub:5]: what is it used for?
these are scripts are copied verbatim from the FSF releases..so wouldn't change them..

**Comment by ipelupessy** [comment]
the last merge looked a bit funny, hope I didn't mess up...

**Comment by rieder** [comment]
> the last merge looked a bit funny, hope I didn't mess up...

looks ok to me

**Comment by rieder** [comment]
where does config.guess originate?

**Comment by rieder** [comment]
and config.sub?

**Comment by ipelupessy** [comment]
if i am not mistaken (it should match):
https://git.savannah.gnu.org/cgit/config.git/plain/config.sub
(mentioned in the file)
but it could also have been copied from some other project

**Comment by rieder** [comment]
I see. Slightly worried about license issues here (I'm no expert at that) but I guess it may be ok.

File: config.sub (hunk 3 above)
Comment (rieder) [reply config.sub:5]: ok

**Comment by rieder** [comment]
ok to merge

**Pull request merged**